packages/dev/serializers/src/USD/USDSerializer.ts

```diff
@@ -93,11 +93,12 @@ export class USDExport {
                             materials[material.uniqueId] = material;
                         }
 
-                        let noopNode: boolean = false;
-                        if (mesh.parent && isNoopNode(mesh.parent, false)) {
+                        let noopNode = mesh.parent && isNoopNode(mesh.parent, scene.useRightHandedSystem) && !scene.useRightHandedSystem;

```

Comment:
There's still this part of my comment to deal with:

> Also, isNoopNode should be called on the root node of the mesh's parent hierarchy, and not on mesh.parent.

You need to go up the `mesh.parent` hierarchy until it is `null` / `undefined`, and use the last non-null mesh found as the mesh to pass to `isNoopNode`.